readthedocs/search/utils.py

```diff
@@ -159,3 +159,31 @@ def _indexing_helper(html_objs_qs, wipe=False):
                 index_objects_to_es.delay(**kwargs)
             else:
                 delete_objects_in_es.delay(**kwargs)
+
+
+def _get_hit_score(res):
```

Comment:
You can use `attrgetter` (`from operator import attrgetter`) to use in the `key` param instead of a lambda/custom function.

Reply:
Opened #5936